services/blue/src/api/DatabaseAccess/DbConnectionHolder.cs

```diff
@@ -70,13 +70,13 @@ public IDbCommand CreateCommand()
 
         public async ValueTask DisposeAsync()
         {
-            if (_connection is IAsyncDisposable asyncDisposableConnection)
+            if (_connection is DbConnection dbConnection)
             {
-                await asyncDisposableConnection.DisposeAsync();
+                await dbConnection.DisposeAsync();
             }
```

Comment:
`DbConnection` should be covered by `IAsyncDisposable`, no need for extra `if` clause.

Reply:
True, though it was meant to improve readability: this way you'd see that `DbConnection` is an probable implementation, justifying the `DisposeAsync` implementation.
Maybe having the `if (is DbConnection)` as the first statement would be better.